src/customHooks/useDelayedPasswordMasking/useDelayedPasswordMaskingUtils.ts

```diff
@@ -0,0 +1,130 @@
+import cloneDeep from 'lodash/cloneDeep';
+
+import { CursorRef } from './useDelayedPasswordMasking.types';
+
+/** replace `event.target.value` with custom value */
+export const makeChangeEvent = (
+  e: React.ChangeEvent<HTMLInputElement>,
+  value: string,
+) => {
+  const modifiedEvent = cloneDeep(e);
```

Comment:
Would the code below achieve the same thing?

```suggestion
  const modifiedEvent = { ...e };
```

Reply:
I suppose it depends on how nested the object is

Reply:
Always nice to see less lodash 😌